internal/sqlxt/sqlxt.go

```diff
@@ -0,0 +1 @@
+package sqlxt
```

Comment:
Is this package used?

Reply:
This package is planned to develop in another PR . 
For now we can remove this package

Reply:
Remove the package